src/controllers/bookings.cr

```diff
@@ -81,8 +81,10 @@ class Bookings < Application
     booking.booked_by_name = user.name
 
     # check concurrent bookings don't exceed booking limits
-    booking_limits = check_booking_limits(tenant, booking)
-    render :conflict, json: booking_limits if booking_limits
+    unless query_params["limit_override"]? == "true"
```

Comment:
As per the comment https://github.com/place-technology/suncorp-ntt/issues/175#issuecomment-1068824084
we want to allow a per-user custom limit using this param